components/notebook-controller/controllers/notebook_controller.go

```diff
@@ -74,10 +74,15 @@ type NotebookReconciler struct {
 
 // +kubebuilder:rbac:groups=apps,resources=statefulsets,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=apps,resources=statefulsets/status,verbs=get;update;patch
+// +kubebuilder:rbac:groups=apps,resources=deployments,verbs=get;list;watch;create;update;patch;delete
```

Comment:
Let's remove this, the controller never touches deployments.
I saw the manifests, they must be wrong.